events/2021-kubevirt-summit/proposals/codebase-teams-productivity-meet-at-a-bar.md

```diff
@@ -0,0 +1,31 @@
+# Title
+
+Codebase, teams and productivity meet at a bar
+
+# Abstract
+
+The relationship between a project codebase structure, its contributors
```

Comment:
specialization of its contributors

or

its contributors' scpecialization

Reply:
Done, thanks.